pyproject.toml

```diff
@@ -0,0 +1,116 @@
+[tool.poetry]
+name = "aries_cloudagent"
+version = "0.10.0-rc0"
+description = ""
+authors = ["Your Name <you@example.com>"]
```

Comment:
Is it permitted to just not include an `authors` entry? Or leave it as an empty list? Maybe we should put something like `Hyperledger Aries <aries@lists.hyperledger.org>`?

Reply:
Unfortunately, an author is required. I just added Hyperledger as the author